template.rb

```diff
@@ -84,6 +96,45 @@
 end
 CODE
 
+if yes?("Are you hosting Shipit on Heroku? (y/n)")
+  inject_into_file "Procfile", " -p $PORT", after: "web: bundle exec rails s"
+  inject_into_file "Gemfile", "ruby '2.2.3'", after: "source 'https://rubygems.org'\n"
```

Comment:
I'm on the fence about using the latest Ruby version and the one on the user's computer. It's probably better to let them downgrade if they need to, but it may be confusing... @byroot ?